scripts/tests-integration-stage.sh

```diff
@@ -181,6 +182,7 @@ export ALIASED_COLLECTION="false"
 if [ "$STAGE" == "REST" ]; then test_rest; fi
 if [ "$STAGE" == "WFS" ]; then test_wfs; fi
 if [ "$STAGE" == "CSW" ]; then test_csw; fi
+if [ "$STAGE" == "REST_WKT_GEOMETRIES" ]; then export WKT_GEOMETRIES="true"; test_rest; fi
```

Comment:
the env variable will keep this value for the following tests, no? Is it not a problem?

Reply:
True, I didn't see board effect: I wanted to test the WKT_GEOMETRIES=true with the rest stage only.
However it is not a problem for the other test to have WKT_GEOMETRIES=true. 

I'll set it to false for the other stages to keep the precedent behaviour of the other tests

Reply:
@alainbodiguel it's fixed